.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -1,32 +1,32 @@
-name: Standard Issue Tempalte
+name: Standard Issue Template
 description: Create an Issue
-title: "[Issue]:"
+title: "[Issue]: "
```

Comment:
I'd prefer we categorize in ways other than the title of the issue (we have various flag options, for instance). Let's not prepend the issue title with "[Issue]:".